client/me/purchases/remove-purchase/style.scss

```diff
@@ -19,8 +19,3 @@
 		text-overflow: ellipsis;
 	}
 }
```

Comment:
I can get rid of it, that was part of the code that existed before I touched it. Doesn't look like it's being used currently.